exchange.htm

```diff
@@ -27,6 +27,9 @@ <h3>Centralized exchanges</h3>
             <div class="col-12 col-sm-6 col-md-6 col-lg-6">
                 <h3>Free/Starter Gridcoin</h3>
                 <ul>
+                    <li>
+                        Use the <a href="https://sciencestats.net" rel="nofollow">ScienceStats Proof-of-Work Faucet</a>
```

Comment:
why nofollow?

Reply:
Just habit I guess. No reason to steal SEO juice from gridcoin.us.